common/presenters/table/object-to-table-head.js

```diff
@@ -1,17 +1,61 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
-function objectToTableHead({ head } = {}) {
-  if (!head) {
-    return undefined
+function buildUrl({ key }, query = {}) {
```

Comment:
Why not set the default value for query at the first point of entry for this file: 

```javascript
function objectToTableHead(query = {}) {
  ...
}
```

Then you can destructure here to only include what you need:

```javascript
function buildUrl({ key }, { sortDirection }) {
  ...
}
```